### PR TITLE
Added CBOT

### PR DESCRIPTION
**Pull request by kompowiec**
.txt because .cbot extension doesn't work with game

### CODE REVIEW

File: c/cbot.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+extern void object::New()
+{
+
+message("Hello world");
```
Comment (Richienb) [inline c/cbot.txt:4]:
```suggestion
message("Hello World");
```

**Comment by Richienb** [comment]
@kompowiec

File: c/cbot.txt (hunk 1 above)
Comment (kompowiec) [reply c/cbot.txt:4]: No, looks almost like the javascript alert command

ach, typo right

Comment (Richienb) [reply c/cbot.txt:4]: @kompowiec All the examples have to display the message "Hello World" in title case.

Comment (kompowiec) [reply c/cbot.txt:4]: yeah i noticed there was a typo, next time i will remember.

**Pull request merged**